### PR TITLE
Safety compliance section

### PR DESCRIPTION
**Pull request by KyleMacPherson**
https://trello.com/c/HfcM3Jw2/1422-cases-page-editing-of-safety-and-compliance-section

## Description
Changes to the UI and behaviour of the edit safety and compliance flow for OPSS users.

Note: The safety and compliance flow for TS remains unchanged and we need to support both of these at the current time.

## Checklist:
- [ ] Have you documented your changes in the pull request description?
- [ ] Does the change present any security considerations?
- [ ] Is any gem functionality overloaded? Eg: Devise controller methods being overloaded.
- [ ] Has acceptance criteria been tested by a peer?
- [ ] Has the CHANGELOG been updated? (If change is worth telling users about.)

### General testing (author)
- [ ] Test without JavaScript
- [ ] Test on small screen

### Accessibility testing (author)
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable CSS - does content make sense and appear in a logical order?

### CODE REVIEW

**Review by nasirkhanbeis — APPROVED** [review]
Tested this on the review app, looks good as expected. 

Suggestion - Wonder if want to add this new page with radios in TS creation journey too to make it consistent across add and edit journey.

File: app/assets/stylesheets/opss/_opss-psd.scss [hunk 1]
```diff
@@ -111,7 +111,7 @@
     }
 
     .opss-panels__single {
-        width: 100%;
+        /*width: 100%; */
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-psd.scss:114]:
```suggestion
```

File: app/assets/stylesheets/opss/_opss-psd.scss [hunk 2]
```diff
@@ -188,6 +188,17 @@ ul.opss-radio-list {
...
+    list-style-type: square;
+
+    ol &,
+    ul & {
+        margin-left: govuk-spacing(6);
+        padding-top: govuk-spacing(4);
+        padding-bottom: govuk-spacing(1);
+    }
+}*/
+
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-psd.scss:201]:
```suggestion
```

File: app/assets/stylesheets/opss/_opss-psd.scss [hunk 3]
```diff
@@ -246,6 +257,7 @@ ul.opss-radio-list {
         font-weight: normal;
         color: $govuk-text-colour;
         border: 1px solid $govuk-border-colour;
+        /*float: right;*/
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-psd.scss:260]:
```suggestion
```

File: app/assets/stylesheets/opss/_opss-shared.scss [hunk 4]
```diff
@@ -21,6 +21,9 @@
 
 $std-font-family: "GDS Transport", "HelveticaNeue", "Helvetica Neue", "Arial", "Helvetica", sans-serif;
 
+/*input[type="checkbox"]:disabled, input[type="checkbox"]:disabled + label {color:  govuk-colour("mid-grey"); }
+input[type="checkbox"]:disabled + label {opacity: 1 !important; }*/
+
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-shared.scss:26]:
```suggestion
```

File: app/assets/stylesheets/opss/_opss-shared.scss [hunk 5]
```diff
@@ -366,6 +369,10 @@ samp {
                 width: 36% !important;
                 @include word-wrap;
             }
+
+            /*dt + dd {    //FF dosen't like
+                text-align: center;
+            }*/
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-shared.scss:375]:
```suggestion
```

File: app/assets/stylesheets/opss/_opss-shared.scss [hunk 6]
```diff
@@ -384,6 +391,7 @@ samp {
 
                 span {
                     @include opss-font-size($s: 14px, $l: 1.2);
+                    /*position: relative; top: -1px;*/
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-shared.scss:394]:
```suggestion
```

File: app/assets/stylesheets/opss/_opss-shared.scss [hunk 7]
```diff
@@ -475,6 +483,10 @@ ul.opss-left-nav {
             }
     }
 
+    /*& > li > ul.govuk-list { // 2nd nested ULs
+        padding-top: govuk-spacing(1); margin: govuk-spacing(4) 0 govuk-spacing(2) 0;
+    } */
+
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-shared.scss:489]:
```suggestion
```

File: app/assets/stylesheets/opss/_opss-shared.scss [hunk 8]
```diff
@@ -509,9 +521,14 @@ ul.opss-left-nav {
...
 .opss-border-right {border-right-width: 1px; }
-.opss-border-all {border-width: 1px; }
 .opss-border-none {border-width: 0; }
-
+@include govuk-media-query($from: tablet) {
+    .opss-rounded-corners {border-radius: 5px; }
+    .opss-drop-shadow {box-shadow: 3px 4px 6px 0px #ebebeb; }//2px 2px 10px govuk-colour("mid-grey");
+    .opss-border-all {border-width: 1px; }
+    /*.opss-border-all--padding-sm {padding: govuk-spacing(1); }
+    a.opss-border-all--padding-sm {padding: 7px 13px 8px 11px; }*/
```
Comment (stdstaff) [inline app/assets/stylesheets/opss/_opss-shared.scss:530]:
```suggestion
```

I wonder if we need to write a rake task to pull the latest version of these styles in, and run them through a CSS linter/beautifier? Or maybe we could vendor the stylesheet as a gem 🤔

**Pull request merged**